packages/gated-content/src/__helpers__/mocks.ts

```diff
@@ -133,3 +134,18 @@ export function mockCollectCondition(overrides?: Partial<CollectCondition>): Col
     ...overrides,
   };
 }
+
+export function mockAdvancedContractCondition(
+  overrides?: Partial<AdvancedContractCondition>,
+): AdvancedContractCondition {
+  return {
+    type: ConditionType.ADVANCED_CONTRACT,
+    contract: mockNetworkAddress(),
+    abi: 'function balanceOf(address) external view returns (uint256)',
+    functionName: 'balanceOf',
+    params: [],
+    comparison: ConditionComparisonOperator.EQUAL,
+    value: ':userAddress',
+    ...overrides,
+  };
+}
```

Comment:
I see this helper but not use of it, was a test suite meant to be checked-in?

Reply:
yeah tests are still WIP, will push them soon